egs/blizzard17/tts1/run.sh

```diff
@@ -0,0 +1,237 @@
+#!/bin/bash
+
+# Copyright 2018 Okayama University (Katsuki Inoue)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+backend=pytorch
+stage=-1
+stop_stage=100
+ngpu=1       # number of gpus ("0" uses cpu, otherwise use gpu)
+nj=32        # numebr of parallel jobs
+dumpdir=dump # directory to dump full features
+verbose=0    # verbose option (if set > 0, get more log)
+N=0          # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+seed=1       # random seed number
+resume=""    # the snapshot path to resume (if set empty, no effect)
+
+
+# feature extraction related
+fs=44100      # sampling frequency
+fmax=""       # maximum frequency
+fmin=""       # minimum frequency
+n_mels=80     # number of mel basis
+n_fft=1024    # number of fft points
+n_shift=256   # number of shift points
+win_length="" # window length
+
+# config files
+train_config=conf/train_pytorch_tacotron2.yaml # you can select from conf or conf/tuning.
+                                               # now we support tacotron2, transformer, and fastspeech
+                                               # see more info in the header of each config.
+decode_config=conf/decode.yaml
+
+# decoding related
+model=model.loss.best
+n_average=1 # if > 0, the model averaged with n_average ckpts will be used instead of model.loss.best
+griffin_lim_iters=1000  # the number of iterations of Griffin-Lim
```

Comment:
We changed to use fast librosa Griffin-Lim, 64 is enough.
```suggestion
griffin_lim_iters=64  # the number of iterations of Griffin-Lim
```